contracts/modules/legacy/LegacyGateway.sol

```diff
@@ -51,7 +52,8 @@ contract LegacyGateway is IGateway, MasterAware {
     incidents = ILegacyIncidents(master.getLatestAddress("IC"));
     pool = IPool(master.getLatestAddress("P1"));
     memberRoles = IMemberRoles(master.getLatestAddress("MR"));
-    cover = ICover(master.getLatestAddress("CO"));
+    // TODO: check contract code
```

Comment:
what's this TODO about?

Is there something left to do here?

Reply:
just make sure that CoverMigrator has indeed the "CL" code - can you confirm @danoctavian ?

Reply:
@roxdanila  confirming CL is CoverMigrator in the upgrade:

https://github.com/NexusMutual/smart-contracts/blob/9ab7a560500c441cbd47e0c624eff17bcb657db7/test/fork/migration-v2.js#L410

Reason being as we know the backwards compatibility with armor and any other users of the legacy Claims code path for submitClaim.